test/integration/integration_test.go

```diff
@@ -159,6 +165,16 @@ var _ = Describe("Test messages", Ordered, func() {
 		}, time.Second*5, time.Millisecond*100).Should(BeNil())
 		VerifyMessageBody(record.Data, vehicleName)
 	})
+
+	It("reads data from zmq subscriber", func() {
+		err := connection.WriteMessage(websocket.BinaryMessage, payload)
+		Expect(err).NotTo(HaveOccurred())
+
+		topic, data, err := zmqConsumer.NextMessage()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(data).NotTo(BeNil())
```

Comment:
validate the data instead of being just checking not nil? See other tests using `VerifyMessageBody`